.github/workflows/pr-build.yaml

```diff
@@ -186,6 +185,7 @@ jobs:
             yq -i '.volumes[0].configMap.name = "lcfs-frontend-dev-${{ env.PR_NUMBER }}"' lcfs/charts/lcfs-frontend/values-dev-pr.yaml
             yq -i '.apiBase = "https://lcfs-backend-dev-${{ env.PR_NUMBER }}.apps.silver.devops.gov.bc.ca/api"' lcfs/charts/lcfs-frontend/values-dev-pr.yaml
             yq -i '.keycloak.postLogoutUrl = "https://lcfs-dev-${{ env.PR_NUMBER }}.apps.silver.devops.gov.bc.ca/"' lcfs/charts/lcfs-frontend/values-dev-pr.yaml
+            yq -i '.tfrsBase = "https://tfrs-dev-${{ env.PR_NUMBER }}.apps.silver.devops.gov.bc.ca/"' lcfs/charts/lcfs-frontend/values-dev-pr.yaml
```

Comment:
Since this is the PR number for LCFS, will this work as a PR number for tfrs-dev or is this a placeholder?

Reply:
Nice catch! the TFRS isn't supporting pr build and the pr number would be different than the lcfs pr number. I'll just close this PR without merging.